libs/langchain/langchain/retrievers/self_query/base.py

```diff
@@ -208,6 +208,7 @@ def from_llm(
         chain_kwargs: Optional[Dict] = None,
         enable_limit: bool = False,
         use_original_query: bool = False,
```

Comment:
All new named arguments should be introduced with `*` present, to minimize accidental breaking changes
```suggestion
        use_original_query: bool = False,
        *,
```